zerver/tests/test_signup.py

```diff
@@ -650,7 +650,7 @@ def test_wrong_subdomain_i18n(self) -> None:
 
         with translation.override("de"):
             body = self.get_reset_mail_body("lear")
-            self.assertIn("hat ein neues Passwort", body)
+            self.assertIn("Passwort", body)
```

Comment:
Why does this line change? Does something else in the commit cause it to need to change? That'd be a surprising interaction.

Reply:
At that time taking advice from [CZO discussion ](https://chat.zulip.org/#narrow/stream/58-translation/topic/Changing.20context.20variable.20names.20breaks.20translation/near/1549667),  I fixed the German translation test by removing the comparison string. Now I re-read the conversations and realized I misinterpreted the `German translation` mentioned in that conversation. I shouldn't change the test; instead, I think I should change the related lines in `locale/de/LC_MESSAGES/django.po`. Later on after merge, we should go to Transifex to sync our change with it. 

I have fixed these lines and `django.po` mentioned above. Also mentioned this translation change in the commit message. Thanks for pointing out!